src/System.Security.Cryptography.Xml/src/Resources/SR.Designer.cs

```diff
@@ -0,0 +1,838 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
Please follow the conventions of the other projects and call this Strings.cs. Also please remove unused strings (should be easy as VS intellisense will show which methods have 0 references)